layers/+emacs/tabs/funcs.el

```diff
@@ -0,0 +1,63 @@
+(defcustom tabs-auto-hide nil
+  "If non-nil hide tabs automatically after TABS-AUTO-HIDE-DELAY seconds."
+  :type '(boolean)
+  :group 'centaur-tabs)
+
+(defcustom tabs-auto-hide-delay 2
+  "Tabs auto hide delay in seconds."
+  :type '(float)
+  :group 'centaur-tabs)
+
+(defun tabs-timer-initialize (secs)
+  (setq tabs-timer (run-with-timer secs nil (lambda () (centaur-tabs-local-mode 1)))))
+
+(defun tabs-timer-hide ()
+  (tabs-timer-initialize tabs-auto-hide-delay))
+
+(when tabs-auto-hide
+  (add-hook 'window-setup-hook 'tabs-timer-hide)
+  (add-hook 'find-file-hook 'tabs-timer-hide)
+  (add-hook 'change-major-mode-hook 'tabs-timer-hide))
+
+(defun centaur-tabs-switch-and-hide (arg)
+  (cancel-timer tabs-timer)
+  (centaur-tabs-local-mode 1)
+  ;; (if arg
+  ;;     (centaur-tabs-backward)
+  ;;   (centaur-tabs-forward))
+  (cond ((equal arg 'backward)
```

Comment:
You can simplify the `cond` form by using `pcase` instead.
```elisp
(pcase arg
  ('backward (centaur-tabs-backward))
  ;; etc
  )
```

Reply:
Thanks for the review and the suggestions. Nice to get reminded about the pcase, I probably will remember its existence this time.

But I do not understand the suggestion about setting the  `centaur-tabs-local-mode 1` in the `let` form? I don't remember exactly why I set this, but it looks that I just want to enable the mode, i.e. hide the tabs, before switching the buffer. Probably so that the tabs will be hidden when you switch back to it via e.g. `SPC b b`. This setting should be persistent, instead of enabled only during evaluation of the `let` form.